### DISCUSSION TITLE
Replay: Download button

### DISCUSSION THREAD

**Discussion by veisveer** [monkeytypegame/monkeytype#1743] [Ideas]
-> To save our Favourite/Best Replays
I can think of 2 logical places:
1) Next to the playpauseReplayButton
2) Next to the saveScreenshotButton

Video file format - what's easiest to implement(=
GIF support?

**Comment by unrealapex** [comment]
That would be something cool to have, with this, it'll be easy to prove you are the speediest of typers.
